.github/workflows/env/fe_dev.env

```diff
@@ -60,4 +60,7 @@ REACT_APP_SPID_CIE_ENTITY_ID=xx_servizicie_test
 REACT_APP_SPID_TEST_ENV_ENABLED='true'
 REACT_APP_ONE_TRUST_BASE_URL=https://dev.selfcare.pagopa.it/ot/test
 
-REACT_APP_PAGOPA_HELP_EMAIL=areariservata@assistenza.pagopa.it
\ No newline at end of file
+REACT_APP_PAGOPA_HELP_EMAIL=areariservata@assistenza.pagopa.it
+
+REACT_APP_CLIENT_ID=client_id_test
```

Comment:
Since this is a property for OI login flow, maybe we can rename this in a more semantic way -> `REACT_APP_OI_CLIENT_ID`